src/sql-pretty/src/doc.rs

```diff
@@ -35,7 +35,7 @@ fn doc_display_pass<'a, T: AstDisplay>(v: &T) -> RcDoc<'a, ()> {
     RcDoc::text(v.to_ast_string())
 }
 
-pub(crate) fn doc_create_view(v: &CreateViewStatement<Raw>) -> RcDoc {
+pub(crate) fn doc_create_view<T: AstInfo>(v: &CreateViewStatement<T>) -> RcDoc {
```

Comment:
this `AstInfo` change seems unrelated to exporting as WASM, but just wanted to confirm